Code/GraphMol/DetermineBonds/DetermineBonds.cpp

```diff
@@ -0,0 +1,422 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "DetermineBonds.h"
+#include <GraphMol/RDKitBase.h>
+#include <YAeHMOP/EHTTools.h>
```

Comment:
YAeHMOP should be moved from Contrib to Code I think.

Reply:
Or maybe just the EHTTools.

Reply:
Since this isn't functionality that everyone will need, we added a configuration option to allow xyz2mol to be disabled rather than add that dependency for everyone who builds the RDKit

Reply:
I see it very likely that I will campaign to put it into the PDBParser, but not for this release at least.

Reply:
> I see it very likely that I will campaign to put it into the PDBParser, but not for this release at least.

I 100% agree that it should be an option for the PDBParser... but as you say, not for this release